### PR TITLE
Changes to accommodate internal bacon framework testing

### PR DESCRIPTION
**Pull request by senthilkumaran-okta**
This could be tested and merged via bacon.

### CODE REVIEW

**Comment by senthilkumaran-okta** [comment]
Review request @tthompson-okta , @tsecor-okta , @kelvinzhu-okta

File: scripts/build_and_test.sh [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 # Use the 1.7 JDK
-JAVA_HOME=${JAVA7_HOME}
-export PATH=${PATH}:${JAVA_HOME}/bin
+export PATH=${PATH}:${JAVA7_HOME}/bin
+
+MAVEN=${OKTA_HOME}/thirdparty/maven-current/bin/mvn
+MAVEN_SETTINGS=${OKTA_HOME}/jenkins-tasks/data/maven/settings.xml
 
-# Define the location of artifactory
-export ARTIFACTORY_URL=https://artifacts.aue1d.saasure.com/artifactory
```
Comment (tthompson-okta) [inline scripts/build_and_test.sh]: :+1: Internal server URLs should definitely not be in the public repo.

**Comment by tthompson-okta** [comment]
LGTM. :ship:

**Comment by senthilkumaran-okta** [comment]
@tsecor-okta : Yes. The log capturing bash stanza is included as part of the parent script.
https://github.com/okta/jenkins-tasks/blob/570dd0e36615bd85d2f1eaa644dd9bda046cf5c9/jobs/generic-build-and-test.sh#L5

(Also, we have update the jenkins job on which this project is run with latest aperture artifact. The ones running on generic.build.and.test.sk show the logs due to latest aperture artifact in them).

**Pull request closed without merging**